src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -52,6 +52,8 @@ internal static class Constants
         public const string OSXRootPath = "~/";
         public const string ManagedDependencyConfigPropertyName = "managedDependency";
         public const string PowerShellWorkerDefaultVersion = "~6";
+        public const string AuthLevelErrorMessage = "Unable to configure Authorization level, the selected template does not use Http Trigger";
```

Comment:
Super Nit: `Unable to configure Authorization level.` Full stop, instead of comma :)

Reply:
updated